config/routes.rb

```diff
@@ -1,4 +1,7 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
-  root 'welcome#index'
+  # root 'welcome#index'
+  root to: 'landing_page#index'
+  post '/quotes', to: 'quotes#search'
```

Comment:
I don't see where this post to quotes#search is being used. I don't think you need this in your routes.